src/main/java/org/jabref/logic/xmp/XmpUtilReader.java

```diff
@@ -104,7 +104,7 @@ private XmpUtilReader() {
      * <p/>
      *
      *
-     * @return empty Optional if no metadata has been found
+     * @return empty Optional if no metadata has been found, or cannot properly find start or end tag in metadata
```

Comment:
It returns an empty list, not an optional. Please adjust thar javadoc comment 
Otherwise looks good

Reply:
Got it